### PR TITLE
updated ngs.json to accommodate scRNA-Seq data

### PR DESCRIPTION
**Pull request by sgosline**
Very rough start, works on issue #339

### CODE REVIEW

**Comment by jaeddy** [comment]
Some of new terms are not independent of existing terms - for example, TruSeq has kits for polyA enrichment OR ribo depletion. Will multiple values be allowed, or do we need to resolve the overlap?

**Comment by sgosline** [comment]
No multiple values will not be allowed, this didn't come up as an issue. We considered just naming the entire kit. Also, you should come to the annotations meeting!!!

File: synapseAnnotations/data/ngs.json [hunk 1]
```diff
@@ -108,7 +108,66 @@
...
+        "description" : "",
+        "source" : ""
+      }
+    ,{
+        "value" : "Smart-seq2",
+        "description" : "",
+        "source" : ""
+      }
+     ,{
+        "value" : "Tru-Seq",
```
[the remark this answers is not in the document]
Comment (sgosline) [reply synapseAnnotations/data/ngs.json]: 🇨🇦

**Comment by sgosline** [comment]
Discussion should be brief on 3/6, but leaving it there just to be sure.

File: synapseAnnotations/data/ngs.json [hunk 2]
```diff
@@ -109,6 +109,130 @@
         "description": "",
         "source": ""
       }
+   ]
+  },
+  {
+    "name": "libraryPrepMethod",
+    "description": " method by which library was prepared",
```
Comment (kdaily) [inline synapseAnnotations/data/ngs.json]: extra space, capital 'M'

File: synapseAnnotations/data/ngs.json [hunk 3]
```diff
@@ -109,6 +109,130 @@
         "description": "",
         "source": ""
       }
+   ]
+  },
+  {
+    "name": "libraryPrepMethod",
```
Comment (kdaily) [inline synapseAnnotations/data/ngs.json]: `libraryPreparationMethod`

File: synapseAnnotations/data/ngs.json [hunk 4]
```diff
@@ -109,6 +109,130 @@
...
+    "description": " method by which library was prepared",
+    "columnType":"STRING",
+    "maximumSize": 256,
+    "enumValues" : [
+      {
+        "value" : "10x",
```
Comment (kdaily) [inline synapseAnnotations/data/ngs.json]: 10x Genomics? isn't this from a specific company?

Comment (sgosline) [reply synapseAnnotations/data/ngs.json]: It's what HCA has, so keeping it.

**Comment by sgosline** [comment]
@kdaily can you review now that I've updated the spacing?

**Pull request merged**